### PR TITLE
landing page added

### PR DESCRIPTION
**Pull request by parulgupta2001**
landing page

### CODE REVIEW

File: my-app/src/App.js [hunk 1]
```diff
@@ -8,6 +15,7 @@ function App() {
     <div className="App">
       <Navbar />
       <Routes>
+        <Route path="/product" element={<ProductListing />} />
```
Comment (MonikaJhingan) [inline my-app/src/App.js:18]: Routes nicely set.

File: my-app/src/frontend/pages/home/home.css [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+}
+
+.stacked-list-item {
+    border:solid 1px var(--color-two);
+    width:20rem;
+    height:4rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--border-color);
```
Comment (MonikaJhingan) [inline my-app/src/frontend/pages/home/home.css:13]: Use of var for colors and use of rems nicely done.

File: my-app/src/frontend/pages/signup/Signup.js [hunk 3]
```diff
@@ -23,7 +22,7 @@ export function Signup() {
       console.log(response.data.encodedToken);
       localStorage.setItem("token", response.data.encodedToken);
       authDispatch({ type: "TOKEN", payload: response.data.encodedToken });
-      navigate("/");
+      navigate("/product");
     } catch (error) {
       console.log(error);
```
Comment (MonikaJhingan) [inline my-app/src/frontend/pages/signup/Signup.js:27]: Throw catch error handling nicely done.

File: my-app/src/frontend/pages/signup/Signup.js [hunk 4]
```diff
@@ -23,7 +22,7 @@ export function Signup() {
       console.log(response.data.encodedToken);
```
Comment (OmKakatkar) [inline my-app/src/frontend/pages/signup/Signup.js:22]: remove the console if not required

**Review by OmKakatkar — APPROVED** [review]
Good variable names
Code is modular

**Review by MonikaJhingan — APPROVED** [review]
The code is good but please indent it a little more to increase code clarity rest is perfectly done.

File: my-app/src/frontend/pages/signup/Signup.js (hunk 3 above)
Comment (shristabaruah) [inline my-app/src/frontend/pages/signup/Signup.js:27]: you  can use `console.error()` for handling errors.

**Review by shristabaruah — APPROVED** [review]
suggestion:
you can use prettier for formating your code.
 Rest looks good

**Review by prachi-sahani — COMMENTED** [review]
- Code is clean and easy to understand
- Good practices have been followed
- Liked the use of barrel imports and try catch blocks

suggestions
- remove console.log

Good to merge

**Pull request merged**